### ISSUE TITLE
system/bin/filesync via shell not working?

### ISSUE THREAD

**Issue by JLNNN** [contao/core#8718]
Everytime I tried
```php
php system/bin/filesync
```
I didn't get any answer from the shell and my database is still empty. Any ideas?

Thank you in advance :)

**Comment by leofeyer** [comment]
Please refer to the [forum](https://community.contao.org/) for this kind of questions. The issue tracker is meant for reporting reproducible bugs in the code base and I cannot reproduce your problem on my system.

<img width="681" alt="" src="https://cloud.githubusercontent.com/assets/1192057/26601930/18bf1f4e-4582-11e7-892f-1d9b5febb659.png">

**Comment by JLNNN** [comment]
Well after a little bit of "echo 123;"-ing, I realized that in system/initialize.php the last line that's executed is 242 (System::importStatic($callback[0])->{$callback[1]}();).

Btw I'm using Contao 3.5.27. I tried the same command on another Contao installation and it works, so it has to be something on that specific installation. I guess it's the amount of files - around 40000 images in files/ folder.

Whenever I try to sync the files via backend I get an error 500 and the site becomes unavailable. At least there's a file in system/tmp/d03a11044f190b5ec548a61f0682aec8 that shows me updated and unchanged files. But this kind of file is not created when I use the php command over shell.

Is there any way I can provide you some more information? If you got a few minutes, I can give you shell access if that helps.

Greetings from Germany
- JLNNN
